kclvm/driver/src/lib.rs

```diff
@@ -22,24 +23,54 @@ use std::{
 };
 use walkdir::WalkDir;
 
+/// Expand the file pattern to a list of files.
+pub fn expand_if_file_pattern(file_pattern: String) -> Result<Vec<String>, String> {
```

Comment:
I suggest using `impl ToString` instead of `String` type as the input and output parameters for functions in future code, but this is only a suggestion and you do not need to modify it in this PR.